src/tokenize.js

```diff
@@ -548,13 +548,38 @@ pp.readString = function(quote) {
 
 // Reads template string tokens.
 
+const INVALID_TEMPLATE_ESCAPE_ERROR = {}
+
+pp.tryReadTemplateToken = function() {
```

Comment:
I'm not very familiar with the tokenizer logic -- is the return value used for anything?